deps/llvm.mk

```diff
@@ -175,6 +175,19 @@ LLVM_CPPFLAGS += -flto
 LLVM_LDFLAGS += -flto
 endif # LLVM_LTO
 
+ifeq ($(BUILD_CUSTOM_LIBCXX),1)
+LLVM_LDFLAGS += -Wl,-R$(build_libdir)
+LLVM_CXXFLAGS += -I$(build_includedir)
+LLVM_CPPFLAGS += -I$(build_includedir)
+LLVM_CFLAGS += -I$(build_includedir)
+LLVM_LIBCXX_FLAGS := -lc++ -lc++abi
```

Comment:
Since these are just `LDFLAGS`, can we stick them onto `LLVM_LDFLAGS` right here?  If you don't want to do that, then let's named them `LLVM_LIBCXX_LDFLAGS` to make it explicit that these are linker flags.